altiProfilAdmin/locales/en_US/admin.UTF-8.properties

```diff
@@ -18,4 +18,8 @@ form.altiProfileTable.label=Database table
 form.srid.label=SRID
 form.ignServiceKey.label=IGN key
 form.ignServiceUrl.label=IGN service URL
-form.altiresolution.label=Resolution
\ No newline at end of file
+form.altiresolution.label=Resolution
+form.connection_check.label=Table queryable ?
```

Comment:
According to HIG guidelines, we do not need a question mark.

What about something like ?

```
Connection status         ✓
```
```
Connection status        Error while connecting to the table (see the log)
```